vllm/engine/async_llm_engine.py

```diff
@@ -1156,7 +1157,13 @@ def remove_logger(self, logger_name: str) -> None:
         self.engine.remove_logger(logger_name=logger_name)
 
     async def start_profile(self) -> None:
-        self.engine.model_executor._run_workers("start_profile")
+        if isinstance(self.engine.model_executor, GPUExecutorAsync):
```

Comment:
type()?

Also could you add a simple comment about why using type instead of isinstnace, as isinstance is more preferable in general

Reply:
[6801e44](https://github.com/vllm-project/vllm/pull/8354/commits/6801e4448d8a3dcc975d175e2dfc8537546ccfee)